script/build.go

```diff
@@ -139,8 +145,8 @@ func buildCommand(dir, buildos, buildarch string) error {
 
 	args := make([]string, 1, 6)
 	args[0] = "build"
-	if len(LdFlag) > 0 {
-		args = append(args, "-ldflags", LdFlag)
+	if len(LdFlags) > 0 {
+		args = append(args, "-ldflags", strings.Join(LdFlags, " "))
```

Comment:
The manual joining into 1 arg could possibly have some quoting issues in the future, I suggest instead:

```
args = append(args, "-ldflags")
args = append(args, LdFlags...)
```